dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops

```diff
@@ -1624,3 +1624,14 @@ uri.has.backslash.character=URI has backslash character; encoded [%5C] or not.
 multipart.file.count.max.exceeded=CWWWC0006E: The request contained more files for upload than is permitted. The maximum number of files is [{0}].
 multipart.file.count.max.exceeded.explanation=The number of files that are uploaded by the multipart/form-data request exceeded the set limit.
 multipart.file.count.max.exceeded.useraction=Increase the maximum number of files that can be uploaded per request by setting the maxFileCount webContainer custom property in the server.xml file. The default value is 5000. For unlimited file upload, set the value to -1.
+
+cannot.sendRedirect.response.already.committed=CWWWC0007E: The response sendRedirect is not allowed after the response is committed.
+cannot.sendRedirect.response.already.committed.explanation=When a response is committed, it cannot send a redirect.
+cannot.sendRedirect.response.already.committed.useraction=Verify the sendRedirect logic to make sure that the response is not committed, either manually or automatically, before the sendRedirect is called.
+
+redirect.location.cannot.be.null=CWWWC0008E: The redirect location cannot be null.
+redirect.location.cannot.be.null.explanation=The redirect location URL needs to be a relative or an absolute URL. 
+redirect.location.cannot.be.null.useraction=Update the redirect location by specifying a URL value that is not empty. The URL can be either relative or absolute.
+
+sendRedirect.hyper.text.url=<html><body><p>Redirecting to <a href="{0}"></a></p></body></html>
+
```

Comment:
Phu, earlier in this file we list the following:

> # Since servlet 4.0.  Use the registered and reserved prefix CWWWC from now on.  
> # 0001 - 0199 for Base
> # 0200 - 0299 for Servlet 3.0
> # 0300 - 0399 for Servlet 3.1
> # 0400 - 0499 for Servlet 4

We should likely update this for later servlet versions to ensure we're following the numbering sequence we've defined and make any necessary updates for Servlet 6.1.

Reply:
Let's discuss once you've taken a look.

Reply:
That was the original plan.  However, these are actually a big range and there is less and less new messages for each releases/services; so it will be a waste of numbers if we go with the original plan.  We are still at a single digit number! (CWWWC0008E with this change)

Also, if you look at the base Messages.nlsprops, we add new message at the end of the file and increasing the current message int number.

I think it should be fine as long as we use the registered prefix `CWWWC` and keep the number sequentially; that way we don't have any gap in the number and also don't have to hunt down the last usage of the servlet version.

Reply:
Ok so it looks like we stopped following that pattern pretty quickly. I only see one that isn't using a base WC number which would be: `CWWWC0400`. Servlet 6.0 used: `CWWWC0005I`.

How do you feel about just removing that comment so we don't cause further confusion in the future? What about servlet-4.0 would we just continue on like we are and ensure that no one uses `CWWWC0400` again?